packages/ember-glimmer/lib/syntax/curly-component.js

```diff
@@ -188,7 +188,6 @@ class CurlyComponentManager {
     aliasIdToElementId(args, props);
 
     props.parentView = parentView;
-    props.renderer = parentView.renderer;
```

Comment:
The last failure is because I removed this code. It seems like engine components are not getting these injections setup: https://github.com/emberjs/ember.js/blob/master/packages/ember-application/lib/system/application-instance.js#L291-L297 (maybe the rest of things in that block too?)

I can add this back, but it seems better to just fix the injection. @rwjblue @dgeb can you halp?